admin/src/app/produtores/page.tsx

```diff
@@ -1,5 +1,49 @@
+"use client";
+
+import { useEffect, useState } from "react";
+
+import IProducerTable from "../components/ProducerTable";
 import Title from "../components/Title";
 
+import { getFarms } from "@admin/_actions/farm/get-farms";
+
+import SearchInput from "@shared/components/SearchInput";
+import { useDebounce } from "@shared/hooks/useDebounce";
+import { useHandleError } from "@shared/hooks/useHandleError"
+import { IProducer } from "@shared/interfaces/user";
```

Comment:
Interfaces com o prefixo I não serão mais utilizadas após o merge do PR #169. Será necessário atualizar quando ele for aprovado.